File: SDKs/Aspose.Imaging_Cloud_SDK_for_Java/src/main/java/com/aspose/imaging/model/BmpProperties.java
package com.aspose.imaging.model;

public class BmpProperties {
  private String Compression = null;
  /**
         * getCompression
         * Gets String
         * @return Compression
         */
  public String getCompression() {
    return Compression;
  }

        /**
         * setCompression
         * Sets String
         * @param Compression String
         */
  public void setCompression(String Compression) {
    this.Compression = Compression;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class BmpProperties {\n");
    sb.append("  Compression: ").append(Compression).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Imaging_Cloud_SDK_for_CSharp/src/Com/Aspose/Imaging/Model/BmpProperties.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Imaging.Model {
  public class BmpProperties {
    public string Compression { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class BmpProperties {\n");
      sb.Append("  Compression: ").Append(Compression).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
